packages/application-generic/src/usecases/get-decrypted-integrations/get-decrypted-integrations.usecase.ts

```diff
@@ -18,10 +18,14 @@ export class GetDecryptedIntegrations {
   async execute(
     command: GetDecryptedIntegrationsCommand
   ): Promise<IntegrationEntity[]> {
-    const query: Partial<IntegrationEntity> & { _environmentId: string } = {
-      _environmentId: command.environmentId,
+    const query: Partial<IntegrationEntity> & { _organizationId: string } = {
+      _organizationId: command.organizationId,
     };
 
+    if (command.environmentId) {
+      query._environmentId = command.environmentId;
+    }
```

Comment:
We give the option to decrypt the integrations:
- all of the organization at once
- all of the specific environment of an organization with this condition